Eastern-Research-Group/expert-query
# Feature/206 api key signup

## Related Issues:
* [EQ-206](https://jira.epa.gov/browse/EQ-206)

## Main Changes:
* Added a page for users to signup for a API key.
  * There are links to this page on the API Documentation page, under the `x-api-key` parameter of each endpoint. However, this and other links in the Swagger UI won't route properly when running on `localhost`, since they're hardcoded into the JSON spec file. If we want, we could transform these URLs before sending the file from the server, but it may not be worth it.

## Steps To Test:
1. Go to http://localhost:3000/api-key-signup.
2. Fill in the required fields, and confirm that an API key is generated successfully (you should receive a confirmation email).
